### PR TITLE
Staff views

### LINKED ISSUE #145: update styling of staff view
http://pulitzercenter.org/about-us/who-we-are

We'll plan to keep all the fields the same, and just apply the new design. Keep the row-list style (as opposed to implementing a grid style).

### PR DESCRIPTION
**Pull request by jselt**
pull
revert
cc all

Go to

http://pulitzer.dev/about-us/who-we-are
http://pulitzer.dev/about-us/who-we-are/student-fellows
http://pulitzer.dev/about-us/who-we-are/reporters
http://pulitzer.dev/about-us/who-we-are/interns

Should be decently formatted lists

Staff fellows should have links to view by year in the right column

Reporters have a search by name filter

closes #145

### CODE REVIEW

File: sites/all/modules/custom/pc_views/pc_views.views_default.inc [hunk 1]
```diff
@@ -2343,9 +2450,9 @@ function pc_views_views_default_views() {
   $handler->display->display_options['fields']['field_image']['label'] = '';
   $handler->display->display_options['fields']['field_image']['element_label_colon'] = FALSE;
   $handler->display->display_options['fields']['field_image']['click_sort_column'] = 'fid';
-  $handler->display->display_options['fields']['field_image']['type'] = 'file_rendered';
+  $handler->display->display_options['fields']['field_image']['type'] = 'image_preset_field_image_preset';
   $handler->display->display_options['fields']['field_image']['settings'] = array(
-    'file_view_mode' => 'default',
+    'image_style' => 'node_images_768x510',
```
Comment (xaqrox) [inline sites/all/modules/custom/pc_views/pc_views.views_default.inc]: Was this intentional?

Comment (jselt) [inline sites/all/modules/custom/pc_views/pc_views.views_default.inc]: No - I thought I changed it back

Comment (jselt) [inline sites/all/modules/custom/pc_views/pc_views.views_default.inc]: is that in the paragraphs view?

**Comment by xaqrox** [comment]
Don't see this
> Staff fellows should have links to view by year in the right column

**Pull request merged**